crypto-ffi/bindings/swift/BuildSettings.xcconfig

```diff
@@ -19,4 +19,7 @@
 // Configuration settings file format documentation can be found at:
 // https://help.apple.com/xcode/#/dev745c5c974
 
-CURRENT_PROJECT_VERSION = 11.0.0
+MARKETING_VERSION=4.2.0
+CURRENT_PROJECT_VERSION=1.0
```

Comment:
`CURRENT_PROJECT_VERSION` needs to be in sync with `MARKETING_VERSION`, per the formula in `update-versions.sh`.